src/Microsoft.OData.Core/UriParser/Aggregation/ApplyClause.cs

```diff
@@ -72,12 +81,21 @@ internal string GetContextUri()
 
         internal List<string> GetLastAggregatedPropertyNames()
         {
-            if (lastAggregateExpressions != null)
+            if (lastAggregateExpressions == null && lastComputeExpressions == null)
             {
-                return lastAggregateExpressions.Select(statement => statement.Alias).ToList();
+                return null;
             }
 
-            return null;
+            List<string> result = new List<string>();
+            if (lastAggregateExpressions != null)
+            {
+                result.AddRange(lastAggregateExpressions.Select(statement => statement.Alias));
+            }
+            if (lastComputeExpressions != null)
+            {
+                result.AddRange(lastComputeExpressions.Select(statement => statement.Alias));

```

Comment:
>result.AddRange(lastComputeExpressions.Select(statement => statement.Alias)); [](start = 16, length = 77)

Can there be duplicates/conflicts when you do 
```
groupby((Type), aggregate($count as Count, Value with sum as Value))/compute(Value = Value div Count)
```

Reply:
Good question and we have the same issue with aggregate. Not going to fix it as part of this PR